content/02.main-text.md

```diff
@@ -20,8 +20,54 @@ Although we requested that some authors participate for their specific expertise
 `TODO: confirm "most"`
 To coordinate this effort, we developed a manuscript writing process using the Markdown language, the GitHub software development platform [@url:https://github.com/greenelab/deep-review/], and our new Manubot tool [@url:https://github.com/greenelab/manubot-rootstock; @url:https://github.com/greenelab/manubot] for automating manuscript generation.
 
-
-### Collaborative writing and reviewing
+### Manubot
+
+We developed Manubot, a system for writing scholarly manuscripts via GitHub.
+With Manubot, manuscripts are written as plain-text markdown files, which is well suited for version control using git.
+The markdown standard itself provides limited yet crucial formatting syntax, including the ability to embed images and format text via bold, italics, hyperlinks, headers, inline code, codeblocks, blockquotes, and numbered or bulleted lists.
+In addition, Manubot relies on Pandoc's extended markdown to enable citations, tables, captions, and equations specified using the popular TeX math syntax.
```

Comment:
Introduce Pandoc?  Something like `Manubot relies on Pandoc, software for converting document formats, to extend markdown and enable`